.github/workflows/bump-brew-formula.yaml

```diff
@@ -0,0 +1,23 @@
+on:
+  release:
+    types: [created]
```

Comment:
release.yaml for creating the release page is triggered by 

```
# A new release is triggered by a new tag of the form viewer-VERSION
on:
  push:
    tags:
      - viewer-*
```

Does this need to be the same?  Setting a tag creates the release page.  Does that trigger a release event and trigger this workflow?

Reply:
Yes it'll trigger this workflow once release is created.
Examples - https://github.com/awslabs/aws-build-accumulator/tree/release/.github/workflows

Reply:
One more thing is to be updated. Release should be created using the bot account and not  GitHub actions.  Will update.